src/index.ts

```diff
@@ -67,6 +67,14 @@ let data = [];
 const testData = ['Person 1', 'Person 2', 'Person 3', 'Person 4'];
 
 let imageUrlForSpinner = '';
+const imageUrlInput = document.getElementById('imageUrlInput') as HTMLInputElement;
+
+function imageUrlSetter() {
+  imageUrlForSpinner = imageUrlInput.value;
+}
+
+const button2 = document.getElementById('imageUrlSetter');
```

Comment:
This variable name could be more verbose (clearer) - for example, it could be called `imageUrlSetterButton`, which would enable other devs to immediately know what it is.

Reply:
Sorry for this, I was actually just being cheeky because you left the 'startSpinning' button named 'button' lol. Would you like me to rename this as well?

Reply:
Really good spot! Please do rename that one 😂